Template/azure/azuresqlserver.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
These values need to be passed into the template. They are dynamic. They are variables. I can't see how this will be used to achieve that.

Reply:
this is dummy one which we created when we are using stages in azure pipeline, it is not needed now will remove it